pkgs/development/libraries/zziplib/default.nix

```diff
@@ -21,6 +22,22 @@ stdenv.mkDerivation rec {
     hash = "sha256-Ht3fBgdrTm4mCi5uhgQPNtpGzADoRVOpSuGPsIS6y0Q=";
   };
 
+  patches = [
+    # apply https://github.com/gdraheim/zziplib/pull/113
```

Comment:
Wow, that's awesome!
Is there an expectation of including it in the upstream?

Reply:
Thanks!
My previous PR (on the same topic...) was merged eventually, so, I think, this one should be fine.